client/me/purchases/cancel-privacy-protection/index.jsx

```diff
@@ -162,7 +166,7 @@ const CancelPrivacyProtection = React.createClass( {
 				<Notice status="is-error" showDismiss={ false }>
 					{ error }
 					{ ' ' }
-					{ this.translate( 'Please try again later or {{a}}contact support.{{/a}}', {
+					{ this.props.translate( 'Please try again later or {{a}}contact support.{{/a}}', {
```

Comment:
We could destructure `translate` since we are already using destructuring in this method.